configs/config.d/puppet-enterprise.conf

```diff
@@ -7,7 +7,7 @@
   @type tail
   format none
   path /var/log/pe-httpd/access.log
-  pos_file /var/lib/google-fluentd/pos/puppet-access.pos
+  pos_file /var/lib/google-fluentd/pos/puppet-enterprise-access.pos
```

Comment:
Would it make sense to put `.pos` files in subdirectories under `/var/lib/google-fluentd/pos/` in the future?

Reply:
That makes sense. I just created b/73718190 for it.